src/core/annotation.js

```diff
@@ -4652,6 +4652,9 @@ class FileAttachmentAnnotation extends MarkupAnnotation {
     const name = dict.get("Name");
     this.data.name =
       name instanceof Name ? stringToPDFString(name.name) : "PushPin";
+
+    const fillAlpha = dict.get("ca");
+    this.data.fillAlpha = typeof fillAlpha === "number" ? fillAlpha : null;
```

Comment:
nit: maybe you should validate or clamp the value.

Reply:
Sure; I went with adding more validation, since that seems overall safer.